dev-support/docker/Dockerfile

```diff
@@ -92,6 +92,15 @@ ENV MAVEN_HOME /usr
 ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64
 ENV FINDBUGS_HOME /usr
 
+#######
+# Install SpotBugs 4.1.3
```

Comment:
Any chance we can make this version number a variables so we keep it consistent with the one in the pom.xml?

Reply:
I want to make this version number a variable but I don't know how to do this. The Dockerfile can be called from many places, so I don't think we can use a variable.